bin/legacy/commands/ci.js

```diff
@@ -0,0 +1,6 @@
+const { builder, handler } = require('../../cml/repo/prepare');
```

Comment:
is this meant to be under `/bin/legacy`?

Reply:
Yes, because (1) it's a legacy command and (2) breaks the `cml <noun> <verb>` schema.